rtt_rosparam/src/rtt_rosparam_service.cpp

```diff
@@ -12,6 +12,29 @@
 
 #include <ros/ros.h>
 
+#ifndef DECLARE_ROSPARAM_GETTER_SETTER
+#define DECLARE_ROSPARAM_GETTER_SETTER(function_name,return_type) \
```

Comment:
but then how to declare all the addOperations ? does it support function overloading ?